Add notifications for exceptions (#291)
crate/crate-operator: CHANGES.rst

Unreleased
----------

> * Replaced kopf timeout handling with a decorator ``@crate.timeout()`` to be
>   able to handle timeouts in the operator.

Reviewer: ```suggestion
* Replaced kopf timeout handling with a decorator ``@crate.timeout()`` to be
  able to run code when a timeout happens.
```
